junebug/config.py

```diff
@@ -38,3 +38,7 @@ class JunebugConfig(Config):
     inbound_message_ttl = ConfigInt(
         "Maximum time allowed to reply to messages",
         default=60)
+
+    outbound_message_ttl = ConfigInt(
+        "Maximum time allowed for events to arrive for messages",
```

Comment:
Err, not sure I'm understanding. I think my question was a little vague, was asking if it would be worth mentioning in the description that the ttls are in seconds.